mangaki/mangaki/tests/test_error_pages.py

```diff
@@ -0,0 +1,39 @@
+from django.test import TestCase, Client
+from mangaki.models import Work, Category, Trope, Editor, Studio
+
+
+class ErrorPageTest(TestCase):
+
+    def setUp(self):
+        # FIXME: The defaults for editor and studio in Work requires those to
+        # exist, or else foreign key constraints fail.
+        Editor.objects.create(pk=1)
+        Studio.objects.create(pk=1)
+
+        self.client = Client()
+        anime = Category.objects.get(slug='anime')
+        work = Work.objects.create(
+            title='Initial D',
+            source='Fujiwara Bunta',
+            category=anime
+        )
+        trope = Trope.objects.create(
+            trope='INERTIA DORIFTU ?!',
+            author='Keisuke Takahashi',
+            origin=work
+        )
+        self.work = work
+        self.trope = trope
+
+    def get_404_url(self):
+        return '/does/not/exist/'
```

Comment:
Pas besoin d'en faire une méthode, ou alors, décore-la avec `@property` ou sors-la de la classe en tant que constante `URL_404 = '/does/not/exist`